packages/fuels-programs/src/logs.rs

```diff
@@ -84,6 +88,19 @@ impl LogResult {
 }
 
 impl LogDecoder {
+    pub fn new(log_formatters: HashMap<LogId, LogFormatter>) -> Self {
+        Self {
+            log_formatters,
+            decoder_config: Default::default(),
+        }
+    }
+
+    // Set the `DecoderConfig` to be used when decoding logs
```

Comment:
```suggestion
    // Set the `DecoderConfig` used when decoding logs
```
or we can delete this comment completely.